### PR TITLE
DHFPROD-7887: Remove Explorer landing page

### PR DESCRIPTION
**Pull request by Sanjeevani19**
#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Code passes ESLint tests
- [x] Added Tests
- [x] Run UI tests
  

- ##### Reviewer:

- [x] Reviewed Tests

### CODE REVIEW

File: marklogic-data-hub-central/ui/src/pages/Browse.tsx [hunk 1]
```diff
@@ -321,27 +318,11 @@ const Browse: React.FC<Props> = ({location}) => {
...
-            start: parsedPreferences.start || 1,
-            pageNumber: parsedPreferences.pageNumber || 1,
-            pageLength: parsedPreferences.pageLength,
-            propertiesToDisplay: searchOptions.selectedTableProperties || [],
-            zeroState: parsedPreferences.zeroState,
-            sortOrder: parsedPreferences.sortOrder || [],
-            database: parsedPreferences.database
-          };
-          await setPageQueryOptions(options);
+        if (!parsedPreferences.zeroState) {
```
Comment (xnikhil08) [inline marklogic-data-hub-central/ui/src/pages/Browse.tsx:325]: I still see few zeroState specific conditions like this here. This condition seems irrelevant, now that there is no zeroState page. It would be better if we remove all such references of zeroState based variables and conditions.

**Review by xnikhil08 — CHANGES_REQUESTED** [review]
Everything looks good, except few references to zeroState still available at few places.
Also, the search queries dropdown shouldn't be visible in the graph view, as its already available in the side panel now and we should use the same for all the views eventually. See the screenshot below.

image

**Comment by Sanjeevani19** [comment]
> Everything looks good, except few references to zeroState still available at few places. Also, the search queries dropdown shouldn't be visible in the graph view, as its already available in the side panel now and we should use the same for all the views eventually. See the screenshot below.
> 
> image

@xnikhil08 removed the references to the zero state in the entire code base. Visibility of search queries dropdown is not working in develop either, Please create a separate bug to handle that.

**Review by xnikhil08 — APPROVED** [review]

**Review by briantang — APPROVED** [review]
Looks good!

**Review by xnikhil08 — APPROVED** [review]
